### ISSUE TITLE
Remove the Walking Bus category

### ISSUE THREAD

**Issue by UKChris-osm** [osmlab/name-suggestion-index#10128]
I suggest we delete the [Walking Bus](https://nsi.guide/index.html?t=transit&k=route&v=walking_bus) category from the index, as it currently only holds one item, which doesn't have a Wikidata attached to it, so doesn't appear in the index.

They seem to refer to safe walking routes (mostly for schools) rather than a specific network that will ever be notable enough for a Wikidata item.

**Comment by bhousel** [comment]
I don't think we can easily remove it because the collector script has collected "Pédibus" and the build script keeps wanting to put it back.  Maybe the category should exist only to match the generic words?
